.gitignore

```diff
@@ -0,0 +1,4 @@
+bin
+obj
+.vscode
+BenchmarkDotNet.Artifacts
```

Comment:
Please end files with trailing newline. I see you are using VSCode, I can recommend the following settings: https://stackoverflow.com/a/57974293/3779655.